autoreject/tests/test_autoreject.py

```diff
@@ -88,8 +88,13 @@ def test_autoreject():
         mne.pick_types(epochs.info, meg=False, eeg=True)[::16],
         mne.pick_types(epochs.info, meg=False, eeg=False, eog=True)]
     pick_ch_names = [epochs.ch_names[pp] for pp in pre_picks]
+    bad_ch_names = [epochs.ch_names[ix] for ix in range(len(epochs.ch_names))
+		    if ix not in pre_picks]
+    epochs_wbads = epochs.copy()
```

Comment:
you mean `epochs_with_bads`? It could also be `epochs_without_bads`. Better to be explicit I guess